### PR TITLE
docs: edit inputs-outputs copy and headers

### PR DESCRIPTION
**Pull request by kapunahelewong**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

Update reduces redundancy, clarifies headers, and removes colloquialisms.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

There is some overlap between this and the [component interaction document](https://angular.io/guide/component-interaction).

### CODE REVIEW

File: aio/content/guide/inputs-outputs.md [hunk 1]
```diff
@@ -1,16 +1,15 @@
-# `@Input()` and `@Output()` properties
+# Sharing data between child and parent directives and components
 
-`@Input()` and `@Output()` allow Angular to share data between the parent context
-and child directives or components. An `@Input()` property is writable
-while an `@Output()` property is observable.
+`@Input()` and `@Output()` help you share data between a parent context and child directives or components.
```
Comment (aikidave) [inline aio/content/guide/inputs-outputs.md:3]:
```suggestion
`@Input()` and `@Output()` help you share data between a parent context and child directives or components.
```
Let's keep this sentence close to the heading. We went from ` child and parent directives and components` to `parent context and child directives or components.

Perhaps:

```
# Sharing data between parent and child compoents

A common pattern in Angular is sharing data between a parent component and one or more child components. You can implement this pattern by using the `@Input()` and `@Output()` directives.

File: aio/content/guide/inputs-outputs.md [hunk 2]
```diff
@@ -19,80 +18,50 @@ Consider this example of a child/parent relationship:
...
 
-`@Input()` and `@Output()` act as
-the API, or application programming interface, of the child
-component in that they allow the child to
-communicate with the parent. Think of `@Input()` and `@Output()` like ports
-or doorways&mdash;`@Input()` is the doorway into the component allowing data
-to flow in while `@Output()` is the doorway out of the component, allowing the
-child component to send data out.
+`@Input()` and `@Output()` give a child component a way to communicate with its parent component.
+`@Input()` is like a doorway into the child component that allows data to flow in while `@Output()` allows the child to send data out.
```
Comment (aikidave) [inline aio/content/guide/inputs-outputs.md:24]:
```suggestion
`@Input()` allows a parent component to update data in the child component. Conversely, `@Output() allows the child to send data to a parent component.
```

File: aio/content/guide/inputs-outputs.md [hunk 3]
```diff
@@ -19,80 +18,50 @@ Consider this example of a child/parent relationship:
...
-## How to use `@Input()`
-
-Use the `@Input()` decorator in a child component or directive to let Angular know
-that a property in that component can receive its value from its parent component.
-It helps to remember that the data flow is from the perspective of the
-child component. So an `@Input()` allows data to be input _into_ the
-child component from the parent component.
+## Sending data to a child component
 
+The `@Input()` decorator in a child component or directive signifies that the property can receive its value from its parent component.
```
Comment (aikidave) [inline aio/content/guide/inputs-outputs.md:31]: Nice!

File: aio/content/guide/inputs-outputs.md [hunk 4]
```diff
@@ -19,80 +18,50 @@ Consider this example of a child/parent relationship:
...
 
-
-
-### In the parent
+### Configuring the parent component
 
 The next step is to bind the property in the parent component's template.
 In this example, the parent component template is `app.component.html`.
 
 First, use the child's selector, here `<app-item-detail>`, as a directive within the
```
Comment (aikidave) [inline aio/content/guide/inputs-outputs.md:58]:
```suggestion
1. Use the child's selector, here `<app-item-detail>`, as a directive within the
```

File: aio/content/guide/inputs-outputs.md [hunk 5]
```diff
@@ -19,80 +18,50 @@ Consider this example of a child/parent relationship:
...
-parent component template. Then, use [property binding](guide/property-binding)
-to bind the property in the child to the property of the parent.
+parent component template.
+Then, use [property binding](guide/property-binding) to bind the `item` property in the child to the `currentItem` property of the parent.
```
Comment (aikidave) [inline aio/content/guide/inputs-outputs.md:60]:
```suggestion
2. Use [property binding](guide/property-binding) to bind the `item` property in the child to the `currentItem` property of the parent.
```

File: aio/content/guide/inputs-outputs.md [hunk 6]
```diff
@@ -19,80 +18,50 @@ Consider this example of a child/parent relationship:
...
 
 <code-example path="inputs-outputs/src/app/app.component.html" region="input-parent" header="src/app/app.component.html"></code-example>
 
-Next, in the parent component class, `app.component.ts`, designate a value for `currentItem`:
+Next, in the parent component class, designate a value for `currentItem`:
```
Comment (aikidave) [inline aio/content/guide/inputs-outputs.md:64]:
```suggestion
3. In the parent component class, designate a value for `currentItem`:
```

File: aio/content/guide/inputs-outputs.md [hunk 7]
```diff
@@ -104,85 +73,46 @@ The following diagram shows this structure:
...
-
-</div>
-
-## In the child
-
-This example features an `<input>` where a user can enter a value and click a `<button>` that raises an event. The `EventEmitter` then relays the data to the parent component.
-
-First, be sure to import `Output` and `EventEmitter`
-in the child component class:
+First, import `Output` and `EventEmitter` in the child component class:
```
Comment (aikidave) [inline aio/content/guide/inputs-outputs.md:107]:
```suggestion
1. Import `Output` and `EventEmitter` in the child component class:
```

File: aio/content/guide/inputs-outputs.md [hunk 8]
```diff
@@ -104,85 +73,46 @@ The following diagram shows this structure:
...
 ```js
 import { Output, EventEmitter } from '@angular/core';
 
 ```
 
-Next, still in the child, decorate a property with `@Output()` in the component class.
-The following example `@Output()` is called `newItemEvent` and its type is
-`EventEmitter`, which means it's an event.
-
+Next, still in the component class, decorate a property with `@Output()`.
```
Comment (aikidave) [inline aio/content/guide/inputs-outputs.md:114]:
```suggestion
1. In the component class, decorate a property with `@Output()`.
```

File: aio/content/guide/inputs-outputs.md [hunk 9]
```diff
@@ -191,152 +121,77 @@ The different parts of the above declaration are as follows:
 * `@Output()`&mdash;a decorator function marking the property as a way for data to go from the child to the parent
 * `newItemEvent`&mdash;the name of the `@Output()`
 * `EventEmitter<string>`&mdash;the `@Output()`'s type
-* `new EventEmitter<string>()`&mdash;tells Angular to create a new event emitter and that the data it emits is of type string. The type could be any type, such as `number`, `boolean`, and so on. For more information on `EventEmitter`, see the [EventEmitter API documentation](api/core/EventEmitter).
+* `new EventEmitter<string>()`&mdash;tells Angular to create a new event emitter and that the data it emits is of type string.
+For more information on `EventEmitter`, see the [EventEmitter API documentation](api/core/EventEmitter).
 
 Next, create an `addNewItem()` method in the same component class:
```
Comment (aikidave) [inline aio/content/guide/inputs-outputs.md:127]:
```suggestion
1. Create an `addNewItem()` method in the same component class:
```

File: aio/content/guide/inputs-outputs.md [hunk 10]
```diff
@@ -191,152 +121,77 @@ The different parts of the above declaration are as follows:
...
 
-The `addNewItem()` function uses the `@Output()`, `newItemEvent`,
-to raise an event in which it emits the value the user
-types into the `<input>`. In other words, when
-the user clicks the add button in the UI, the child lets the parent know
-about the event and gives that data to the parent.
+The `addNewItem()` function uses the `@Output()`, `newItemEvent`, to raise an event with the value the user types into the `<input>`.
 
-### In the child's template
+#### Configuring the child's template
```
Comment (aikidave) [inline aio/content/guide/inputs-outputs.md:133]: Do we have to use L4 headings? I'd prefer to keep it to L3.

File: aio/content/guide/inputs-outputs.md [hunk 11]
```diff
@@ -191,152 +121,77 @@ The different parts of the above declaration are as follows:
...
+The `addItem()` method takes an argument in the form of a string and then adds that string to the `items` array.
 
-### In the parent's template
+#### Configuring the parent's template
 
-Next, in the parent's template, bind the parent's
-method to the child's event. Put the child selector, here `<app-item-output>`,
-within the parent component's
-template, `app.component.html`.
+Next, in the parent's template, bind the parent's method to the child's event.
```
Comment (aikidave) [inline aio/content/guide/inputs-outputs.md:156]:
```suggestion
1. In the parent's template, bind the parent's method to the child's event.
```

File: aio/content/guide/inputs-outputs.md [hunk 12]
```diff
@@ -191,152 +121,77 @@ The different parts of the above declaration are as follows:
...
+Put the child selector, here `<app-item-output>`, within the parent component's template, `app.component.html`.
```
Comment (aikidave) [inline aio/content/guide/inputs-outputs.md:157]:
```suggestion
1. Put the child selector, here `<app-item-output>`, within the parent component's template, `app.component.html`.
```

**Review by aikithoughts — APPROVED** [review]
LGTM! Thanks for incorporating the feedback.

Reviewed-for: global-docs-approvers

**Pull request closed without merging**